introduction/java_if_else/java_if_else.java

```diff
@@ -0,0 +1,26 @@
+import java.util.Scanner;
+
+public class Solution {
+  
+  private static final String WEIRD = "Weird";
+  private static final String NOT_WEIRD = "Not Weird";
+  
+  private static String isWeird(int numberToCheck) {
+    if ((numberToCheck % 2 == 1) && (numberToCheck <= 100) && (numberToCheck >= 1)) {
```

Comment:
Remove redundant brackets: arithmetic operations have higher priority than logical ones, so this expression will stay the same without brackets.
+Same below.